projects/cashmere-examples/src/lib/modal-simple/modal-simple-example.component.html

```diff
@@ -1,7 +1,6 @@
 <p>To implement simple modals with minimal code, use <code>alert()</code>, <code>confirm()</code>, or <code>confirmDestructive()</code>:</p>
 
-<pre><code>
-this.modalService.confirm(options)
```

Comment:
@corykon `<pre>` is really strict with newlines 😄 so you were getting an extra line at the top of your codeblock that you didn't need